Python/emscripten_trampoline.c

```diff
@@ -78,6 +78,10 @@ EM_JS(CountArgsFunc, _PyEM_GetCountArgsPtr, (), {
 //     )
 // )
 addOnPreRun(() => {
+    let isIOS = globalThis.navigator && /iPad|iPhone|iPod/.test(navigator.platform);
+    if (isIOS) {
+        return;
```

Comment:
Is there any concern here that this path won't trigger the `Module._PyEM_CountArgsPtr = 0` logic which is what happens in other cases if the GC module can't be loaded (L161-168 of this file)? In terms of consistency, it seems preferable that the iOS shortcut would end up with the same module state as the "couldn't load WASM-GC" code path.

Reply:
Good point. Maybe the best thing to do to would be to extract this to a separate `getPtr()` function.

Reply:
This is addressed.